### PR TITLE
New translation strings

### PR DESCRIPTION
**Pull request by Dxgamer7405**

### CODE REVIEW

**Comment by Dxgamer7405** [comment]
dont merge yet, something happened and now the game cant start idk why

File: Patches/ChatCommandPatch.cs [hunk 1]
```diff
@@ -249,7 +249,8 @@ public static bool Prefix(ChatController __instance)
                         }
                         else Main.SetRoles[targetPc.PlayerId] = roleToSet;
 
-                        Utils.SendMessage("\n", localPlayerId, $"<b>{Utils.ColorString(Main.PlayerColors.TryGetValue(resultId, out var textColor) ? textColor : Color.white, targetPc.GetRealName())}</b>'s role in the next game will be <b><color={Main.roleColors[roleToSet]}>{GetString(roleToSet.ToString())}</color></b>");
+                       var playername = $"<b>{Utils.ColorString(Main.PlayerColors.TryGetValue(resultId, out var textColor) ? textColor : Color.white, targetPc.GetRealName())}</b>";
+                       var rolename = $"<color={Main.roleColors[roleToSet]}> {GetString(roleToSet.ToString())} </color>";
```
Comment (Gurge44) [inline Patches/ChatCommandPatch.cs:253]: Where did Utils.SendMessage() go?

File: Resources/Lang/en_US.json [hunk 2]
```diff
@@ -3543,5 +3543,10 @@
   "ForceEnd": "Aborted",
   "EveryoneDied": "Everyone died",
   "ForceEndText": "Host has aborted the game",
-  "DiscoChangeInterval": "Disco Change Color Interval"
-}
+  "DiscoChangeInterval": "Disco Change Color Interval",
+  "Modifier": "\nModifier: ",
+  "Modifiers": "\nModifiers: ",
```
Comment (Gurge44) [inline Resources/Lang/en_US.json:3548]: \n needs to be \\n

Comment (Gurge44) [reply Resources/Lang/en_US.json:3548]: Double slash n

File: Resources/Lang/en_US.json [hunk 3]
```diff
@@ -3543,5 +3543,10 @@
...
+  "RoleSelected": "<b>The role of {0} in the next game will be{1}</b>",
```
Comment (Gurge44) [inline Resources/Lang/en_US.json:3549]: missing space before {1}

Comment (Dxgamer7405) [reply Resources/Lang/en_US.json:3549]: the space is on code

File: Resources/Lang/en_US.json (hunk 2 above)
Comment (Dxgamer7405) [reply Resources/Lang/en_US.json:3548]: it worked with one slash

File: Patches/ChatCommandPatch.cs (hunk 1 above)
Comment (Dxgamer7405) [reply Patches/ChatCommandPatch.cs:253]: idk, this code was made by ph gaming, but when i tested setrole worked

Comment (Dxgamer7405) [reply Patches/ChatCommandPatch.cs]: forgot to put here

Comment (Dxgamer7405) [reply Patches/ChatCommandPatch.cs]: done

**Comment by Dxgamer7405** [comment]
merge now pls

**Pull request merged**